extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -517,43 +530,47 @@ angular.module('oppia').component('svgFilenameEditor', {
           ctrl.data.savedSvgFileName.length > 0);
       };
 
+      var getStrokeWidth = function() {
+        var size = ctrl.fabricjsOptions.size;
+        // Removes the word "px" from the end of the string.
+        return parseInt(size.substring(0, size.length - 2));
+      };
+
       ctrl.createRect = function() {
         ctrl.canvas.discardActiveObject();
-        var size = ctrl.fabricjsOptions.size;
         var rect = new fabric.Rect({
-          top: 50,
-          left: 50,
+          top: ctrl.defaultTopCoordinate,
+          left: ctrl.defaultLeftCoordinate,
           width: 60,
           height: 70,
```

Comment:
Are these default values? If so, assign and use a descriptive variable name to indicate the same.

Reply:
Done.